modules/files.nix

```diff
@@ -111,6 +116,10 @@ in
           for sourcePath in "$@" ; do
             relativePath="''${sourcePath#$newGenFiles/}"
             targetPath="$HOME/$relativePath"
+            if [[ -e "$targetPath" && -n "$HOME_MANAGER_BACKUP_EXT" ]] ; then
+              backup="''${targetPath}''${HOME_MANAGER_BACKUP_EXT}"
```

Comment:
`backup="$targetPath$HOME_MANAGER_BACKUP_EXT"` should work fine.

Also, shouldn't there be a period between the target path and the extension? I imagine `-b bck` applied to a file `xyz.txt` will product `xyz.txt.bck`?

Reply:
I'd been thinking `-b .bck` for that case, but I'm indifferent as to which approach we take. Always inserting a `.` probably makes more sense.